chrome/iso.css

```diff
@@ -150,6 +150,15 @@
   vertical-align: bottom;
 }
 
+.ic-stats-total-meta {
+  vertical-align: middle;
+}
+
+.ic-stats-average {
+  font-weight: bold;
+  color: black;
```

Comment:
Let's kill the `color` rule here and have the default GitHub `body` color come through (`#24292e`). 

Related: [this is a great tip](https://ianstormtaylor.com/design-tip-never-use-black/).

Reply:
Hey @jasonlong , thanks a lot for sharing this article, it was really interesting to read :slightly_smiling_face: